CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Fix: bug in GTE and LTE operations in query filters, both for GET operations and subscriptions (#2995)
```

Comment:
Suggestion:

```
- Fix: bug in GTE and LTE operations in query filters (q/mq), both for GET operations and subscriptions (#2995)
```

to be more explicit with "query filters".

Btw, is this PR solving also the problem described in https://github.com/telefonicaid/fiware-orion/issues/2998 ?

Reply:
Haven't tried that issue.
It's about `null` values. I have no reason to believe the changes in this PR solves that problem.
I can start with that as soon as this PR in ready and merged

Reply:
Ok, thus #2998 must not be mentionces. However, I think mentioning q/mq (see my suggestion above) would be helpful.

Reply:
(q/mq) fixed in 760377b81